about/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Please make your HTML prettier. Follow that rule https://github.com/hail2u/html-best-practices#other.

Reply:
Leveled